tests/scripts/protect-i18n.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+/**
+ * What is this? With this small JavaScript file we want
+ * to block the upload of modified files in the "locale"
+ * folder to the repository through a git hook generated
+ * with the Husky library.
+ *
+ * Usage: $ node .git-hook/locale.js
+ *
+ * Happy releasing!
+ */
+
+'use strict'
+
+/*
+ * Check Github Action Default environment variables.
+ * More info: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables
+ */
+if (process.env.CI) process.exit()
+
+// Require main dependences.
+const { execSync } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+// Execute Git Diff for get all files with changes.
+const results = execSync('git diff --cached --name-only')
```

Comment:
get diff of all locale folder and exclude EN
```suggestion
const results = execSync('git diff --cached --name-only locale/ :!locale/en')
```

Reply:
Hi @Aissaoui-Ahmed we are controlling that [here](https://github.com/JoseJPR/nodejs.org/blob/feature/git-hook-i18n-translate/tests/scripts/protect-i18n.js#L38). But, do you see any greater benefit in your proposal? :)

Reply:
Hi @JoseJPR if you run `git diff --cached --name-only locale/ ':!locale/en'` return only changes on folder `locale` and exclude `locale/en`
you don't need [regex](https://github.com/JoseJPR/nodejs.org/blob/feature/git-hook-i18n-translate/tests/scripts/protect-i18n.js#L38)